### PR TITLE
Create form_automation.md

### PR DESCRIPTION
**Pull request by sychang**
Template for submitting requests for Workflows forms to be automated

## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [x] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/form_automation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+### Request Information				
+#### State:
+
+#### Opportunity name:
+
+#### Link to original form in Google Drive:
+#### Required form fields:
+- [ ] Fill out column C of the appropriate state ingest spreadsheet with all of the fields that will be required to fill out this form [ME example](https://docs.google.com/spreadsheets/d/1xayFUaB-I8j3Izg6Ip2CmMwgYDDLKNg5ckZuWAz8bsY/edit#gid=1337979536). [ME](https://docs.google.com/spreadsheets/d/1xayFUaB-I8j3Izg6Ip2CmMwgYDDLKNg5ckZuWAz8bsY/edit#gid=1337979536); [ND](https://docs.google.com/spreadsheets/d/1L7kkvOhYGp2HJ38n3Kxut3rDUXrWte9oHO_U_w6x92U/edit#gid=0); [ID](https://docs.google.com/spreadsheets/d/1Fu5VsJZJAIUjNpjSSv3ZiJlgjbq8K9aacQUm6O8MivY/edit#gid=0).
+
+#### Please provide any extra context if necessary:
```
Comment (Catacola) [inline .github/ISSUE_TEMPLATE/form_automation.md:18]: Should we ask about expected timelines or expected impact?

**Review by Catacola — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/form_automation.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+### Request Information				
+#### State:
+
+#### Opportunity name:
+
+#### Link to original form in Google Drive:
+#### Required form fields:
+- [ ] Fill out column C of the appropriate state ingest spreadsheet with all of the fields that will be required to fill out this form [ME example](https://docs.google.com/spreadsheets/d/1xayFUaB-I8j3Izg6Ip2CmMwgYDDLKNg5ckZuWAz8bsY/edit#gid=1337979536). [ME](https://docs.google.com/spreadsheets/d/1xayFUaB-I8j3Izg6Ip2CmMwgYDDLKNg5ckZuWAz8bsY/edit#gid=1337979536); [ND](https://docs.google.com/spreadsheets/d/1L7kkvOhYGp2HJ38n3Kxut3rDUXrWte9oHO_U_w6x92U/edit#gid=0); [ID](https://docs.google.com/spreadsheets/d/1Fu5VsJZJAIUjNpjSSv3ZiJlgjbq8K9aacQUm6O8MivY/edit#gid=0).
```
Comment (danawillow) [inline .github/ISSUE_TEMPLATE/form_automation.md:16]: Looks like you have ME twice here. Also a suggestion:
```suggestion
- [ ] Fill out column C of the appropriate state ingest spreadsheet with all of the fields that will be required to fill out this form. Examples: [ME](https://docs.google.com/spreadsheets/d/1xayFUaB-I8j3Izg6Ip2CmMwgYDDLKNg5ckZuWAz8bsY/edit#gid=1337979536); [ND](https://docs.google.com/spreadsheets/d/1L7kkvOhYGp2HJ38n3Kxut3rDUXrWte9oHO_U_w6x92U/edit#gid=0); [ID](https://docs.google.com/spreadsheets/d/1Fu5VsJZJAIUjNpjSSv3ZiJlgjbq8K9aacQUm6O8MivY/edit#gid=0).
```

File: .github/ISSUE_TEMPLATE/form_automation.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Workflows Autofilled Form request
+about: Request a form to be autofilled for a particular Workflow opportunity.
+title: '[US_XX] Form Automation: '
+labels: 'Team: Polaris, Project: Workflows'
+assignees: 'sychang'
+---
+
+### Request Information
```
Comment (danawillow) [inline .github/ISSUE_TEMPLATE/form_automation.md:9]: nit: there are a few tabs hanging out at the end of this line; may as well get rid of them
```suggestion
### Request Information
```

**Review by danawillow — APPROVED** [review]
Looks good! We can always iterate if we realize we need different information.

**Pull request merged**